project/my-app/src/page/login.js

```diff
@@ -0,0 +1,44 @@
+import {useState} from "react"
+import {useNavigate} from "react-router-dom"
+import { InputField,Lable} from "../Component/styled";
+const LoginPage =()=>{
+
+    const [users,setUsers] =useState(
+        {
+            name: "",
+            email:"",
+            Password:"",
+        }
+    )
+    let navigate = useNavigate();
+    const HandleChange =() =>{
+      setUsers({
+            ...users,
+            name:document.getElementById("name").value,
+            email:document.getElementById("email").value,
```

Comment:
Document.getElement by ID should be here instead of that use onChange function take the targeted value

Reply:
ok sir I will correct it